src/commands/commit.ts

```diff
@@ -61,58 +62,127 @@ ${chalk.grey('——————————————————')}
 ${commitMessage}
 ${chalk.grey('——————————————————')}`
   );
+  
+  const promptUserConfirm = async(commitText: string ) => {
+
+    const isCommitConfirmedByUser = await select({
+      message: 'Confirm the commit message',
+      options: [
+        {value: "yes", label: "Yes"},
+        {value: "no", label: "No"},
+        {value: "edit", label: "Edit"}
+      ]
+      
+    });
 
```

Comment:
I changed isConfirmedByUser from a yes/no prompt to a multi select. I also encapsulated the conditionals into their own function so that they can be called again from the new promptUserEdit function further down.